### PR TITLE
Minor edits to comms

### PR DESCRIPTION
**Pull request by plaindocs**
I already mailed these out, just need a +1 to merge :-p

### CODE REVIEW

File: docs/conf/prague/2022/cfp-email-templates.rst [hunk 1]
```diff
@@ -0,0 +1,301 @@
...
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Subject:
+   WTD {{city}} {{year}} -- speaker logistics
+
+::
+
+   Hi {name},
+
+   Just wanted to drop you all a quick note covering some logistics. Talks are pre-recorded again this year, we'd love to get those recordings from you before {{cfp.video_by}}. We do have a videographer who can help you record remotely, drop us a line if that is something you're interested in. We're finzalizing details on the live Q&A for each talk.
```
Comment (mxsasha) [inline docs/conf/prague/2022/cfp-email-templates.rst]:
```suggestion
   Just wanted to drop you all a quick note covering some logistics. Talks are pre-recorded again this year, we'd love to get those recordings from you before {{cfp.video_by}}. We do have a videographer who can help you record remotely, drop us a line if that is something you're interested in. We're finalizing details on the live Q&A for each talk.
```

File: docs/conf/prague/2022/cfp-email-templates.rst [hunk 2]
```diff
@@ -0,0 +1,301 @@
...
+
+::
+
+   Hi {name},
+
+   Here are the important details you've been waiting for! I'll get into specifics below, but first the important ones:
+
+   ○ we'd love you to upload your recorded talk by the **{{ cfp.video_by }}**, or soon after. If you're likely to need more time, please let me know in advance.
+   ○  we have folks who can help you record online, both in US and EU time zones, if this is of interest just let me know and I'll get a slot booked. [Recording guidelines](https://www.writethedocs.org/conf/{{shortcode}}/{{year}}/talk-recording-guidelines/).
+   ○  as well as the tips in the recording guidelines we've updated the [speaking tips](https://www.writethedocs.org/conf/{{shortcode}}/{{year}}/speaking-tips/) for virtual conferences.
```
Comment (mxsasha) [inline docs/conf/prague/2022/cfp-email-templates.rst]: Nitpick, but this and the previous three bullets should start with a capital, I think? Consistent with next one and with use of period at the end of each point.

**Review by mxsasha — APPROVED** [review]
👍🏻  with two notes inline

**Comment by plaindocs** [comment]
@mxsasha Looks to me like our spellcheck might be broken. Though codespell doesn't pick `finzalizing` up anyway, but there are a couple of others showing up locally

**Pull request merged**